apps/kcdbundle/src/pages/buy/email.tsx

```diff
@@ -57,19 +57,26 @@ const BuyEmailForm: React.FC<BuyEmailFormProps> = ({
       .post('/api/users', {email})
       .then(({data}) => data)
 
-    console.debug(purchases)
+    const stripe_customer_id = get(
+      find(purchases, (purchase) => purchase.stripe_customer_id),
+      'stripe_customer_id',
+    )
 
     const availableToUpgrade = sellables.filter((sellable) => {
-      return !purchases.includes(sellable.site)
+      return !find(purchases, (purchase: any) => {
+        return purchase.site === sellable.site
+      })
     })
 
-    console.debug(purchases)
-
     if (isEmpty(availableToUpgrade)) {
       throw new Error('no upgrade available')
     }
 
-    return createCheckoutSession(availableToUpgrade).then(async (data) => {
+    return createCheckoutSession(
+      availableToUpgrade,
+      email,
```

Comment:
we want to send the email regardless of stripe customer so it can be pre-filled with whatever they enter here (validated)